### PR TITLE
Grid column menu icons in rtl

### PR DESCRIPTION
**Pull request by TeyaVes**
targets: https://github.com/telerik/kendo-themes/issues/709

related to: https://github.com/telerik/kendo/pull/9678

### CODE REVIEW

File: tests/visual/grid-column-menu-left.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
+<!doctype html>
```
Comment (PreslavKozovski) [inline tests/visual/grid-column-menu-left.html]: Change the file name to "...-rtl", not "left"

File: tests/visual/grid-column-menu-left.html [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Grid Column Menu</title>
```
Comment (PreslavKozovski) [inline tests/visual/grid-column-menu-left.html]: Change the title

File: packages/default/scss/menu/_layout.scss [hunk 3]
```diff
@@ -116,6 +116,14 @@ $image-spacing: 8px !default;
         .k-item {
             display: block;
             border-color: inherit;
+
+            .k-icon {
+                margin-right: $image-spacing;
```
Comment (PreslavKozovski) [inline packages/default/scss/menu/_layout.scss]: If the `margin-right` style is inside "menu" file. Why the relevant RTL styles are in the "grid" file?

Comment (TeyaVes) [reply packages/default/scss/menu/_layout.scss]: The margin-left and right in the 'grid' file is resetting the margins in RTL mode for the column menu within the grid, which are set as:

    .k-columnmenu-item {
        padding: $nav-item-padding-y $nav-item-padding-x;
        cursor: pointer;

        > .k-icon {
            margin-right: $icon-spacing;
        }
    }
  
While the styles set in the 'menu' file are targeting the menu outside of the grid.

**Pull request merged**